src/main/java/yevhenii/lostfilmdemo/repository/impl/JooqTVSeriesRepository.java

```diff
@@ -0,0 +1,70 @@
+package yevhenii.lostfilmdemo.repository.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.jooq.DSLContext;
+import org.jooq.Record;
+import yevhenii.lostfilmdemo.jooq.generated.tables.records.TvSeriesRecord;
+import yevhenii.lostfilmdemo.repository.JooqRepository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static yevhenii.lostfilmdemo.jooq.generated.tables.TvSeries.TV_SERIES;
+
+@org.springframework.stereotype.Repository
+@RequiredArgsConstructor
+public class JooqTVSeriesRepository implements JooqRepository<String> {
```

Comment:
no need specify that this is JooqRepository in the name, just TVSeriesRepository, tommorow it can become spring data or something different